default.yaml

```diff
@@ -6,17 +6,24 @@
 printer:
   serial: /home/pi/printer_data/comms/klippy.serial
   baudrate: 250000
-  frame_buffer: /dev/fb0
-  fb_bit_depth: 5
-  fb_chunk_size: 16
-  max_z: 350
+  max_z: 300
   default_lift: 10
   default_up_speed: 3.4
   default_down_speed: 3.4
   default_wait_before_exposure: 2.2
   default_wait_after_exposure: 1.5
   pause_lift: 100
 
+# This section holds fields pertaining to the display used by the printer
+display:
+  frame_buffer: /dev/fb0
+  bit_depth:
```

Comment:
Maybe we can assume 565 as the default? Afaik most of these screen drivers use RGB565.

Reply:
I don't think the Prusa screens do? And regardless, I'd like to keep the option to change it if necessary